nav2_behavior_tree/plugins/control/sequence_with_blackboard_memory.cpp

```diff
@@ -0,0 +1,100 @@
+// Copyright (c) 2019 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "nav2_behavior_tree/plugins/control/sequence_with_blackboard_memory.hpp"
+#include "behaviortree_cpp/action_node.h"
+#include "behaviortree_cpp/bt_factory.h"
+
+namespace nav2_behavior_tree
+{
+
+SequenceWithBlackboardMemoryNode::SequenceWithBlackboardMemoryNode(
+  const std::string & name,
+  const BT::NodeConfiguration & conf)
+: BT::ControlNode::ControlNode(name, conf)
+{
+  setRegistrationID("SequenceWithBlackboardMemory");
```

Comment:
Question: What's this about? We don't currently do this in our BT nodes... should we be for newer versions of BT.CPP?